services/app/app/database.py

```diff
@@ -140,3 +156,31 @@ def weather_set(self, weather: Weather) -> Optional[Weather]:
                 values (%s, %s, %s, %s)
             """, (weather.location, weather.temperature, weather.phrase, weather.on.isoformat()))
         return self.weather_get(weather.location, weather.on)
+
+    def inventory_get(self, inventory_id: Optional[int] = None, player: Optional[int] = None):
+        with self.engine.connect() as connection:
+            if inventory_id is not None:
```

Comment:
This validation doesn't need to happen here if you strengthen the input requirements of `def inventory_get`. You can likely get away with this by removing the `Optional` constraint from the `inventory_id` property.